tools/scripts/build/templateInstallationPackage/scriptPreffix.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I have not noticed that.
No, it is not a valid header.
Please @alexandrelimassantana fix that too.
We have a sample file with header examples that you can copy and paste, the file is called "licenses-header-template". The proper header for .sh files is the one with "#" comment markers